### ISSUE TITLE
NBT in item creator not working

### ISSUE THREAD

**Issue by haohaostudio** [CodeDoctorDE/ItemMods#10]
**Describe the bug**
NBT in item creator not working. Nothing happens on click.

**To Reproduce**
1. Go to item creator
2. Click on "NBT"
3. See error

**Expected behavior**
Open something to allow me to modify its nbt

**Desktop (please complete the following information):**
 - OS:  Windows 10 Java 8

**Comment by CodeDoctorDE** [comment]
Yes, currently the version 1.x doesn't support 1.16. I rewrite the plugrin. it should work with 2.0!
